### DISCUSSION TITLE
Cannot connect to homebrew.bintray.com

### DISCUSSION THREAD

**Discussion by opus68** [Homebrew/discussions#67] [Everyday usage] [answered]
I am trying to run 'brew upgrade', but since yesterday (16 October) brew has been failing to connect to homebrew.bintray.com:

```
    ==> Upgrading libomp 10.0.1 -> 11.0.0
    ==> Downloading https://homebrew.bintray.com/bottles/libomp-11.0.0.catalina.bottle.tar.gz
    #=#=#
    curl: (7) Couldn't connect to server
    Error: Failed to download resource "libomp"
    Download failed: https://homebrew.bintray.com/bottles/libomp-11.0.0.catalina.bottle.tar.gz
```

The same is true for a few other bottles. 

However, I can download (through Cloudfront) the file(s) in questions in my browser with no problems. 

pinging homebrew.bintray.com fails:

```
    ➜  ~ ping -c1 homebrew.bintray.com
    PING dl.bintray.com (52.28.44.12): 56 data bytes

    --- dl.bintray.com ping statistics ---
    1 packets transmitted, 0 packets received, 100.0% packet loss
```
brew doctor says I'm ready to brew as usual.

Can anybody help?

Thanks
Michael

**Comment by SMillerDev** [comment, marked as answer]
This seems like a network problem. It's not related to homebrew
